src/Symfony/Component/Form/Forms.php

```diff
@@ -24,11 +24,12 @@
  * $formFactory = Forms::createFormFactory();
  *
  * $form = $formFactory->createBuilder()
- *     ->add('firstName', 'text')
- *     ->add('lastName', 'text')
- *     ->add('age', 'integer')
- *     ->add('gender', 'choice', array(
- *         'choices' => array('m' => 'Male', 'f' => 'Female'),
+ *     ->add('firstName', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```

Comment:
Sadly that's not true :sob: Worst case: ~66% of WordPress users run PHP 5.4 or lower ([source](https://wordpress.org/about/stats/)). Best case: ~25% of SonataProject users run PHP 5.4 or lower ([source](https://twitter.com/sonataproject/status/699278314817593344)) and ~25% of Packagist/Composer users run PHP 5.4 or lower ([source](https://seld.be/notes/php-versions-stats-2015-edition))